stdlib/os/os.go

```diff
@@ -61,6 +61,7 @@ func init() {
 		py.MustNewMethod("rmdir", rmdir, 0, rmdir_doc),
 		py.MustNewMethod("system", system, 0, "Run shell commands, prints stdout directly to default"),
 		py.MustNewMethod("unsetenv", unsetenv, 0, "Unset (delete) the environment variable named key."),
+		py.MustNewMethod("listdir", listDir, 0, listDir_doc),
```

Comment:
please keep the slice ordered by method name.

(yes, `chdir` should probably be moved up closed to `close`.)

Reply:
Fixed.